server/lib/resolvers/builders/postgres.js

```diff
@@ -7,6 +7,7 @@ function buildPostgresResolver (dataSource, compiledRequestMapping, compiledResp
     return new Promise((resolve, reject) => {
       const dataSourceClient = dataSource.getClient()
       const requestTimeStart = Date.now()
+      info['dataSourceType'] = dataSource.type
```

Comment:
data source type will always be Postgres in postgres.js and InMemory in nedb.js

Reply:
ah, but I see your point now. you just want to get it from the DataSource impl using its type property. that's also ok